File: exercicios em C/q81.cpp
#include <stdio.h>

int main(void){
	int volume, i, n, aresta, soma;
	volume = 0;
	soma = 0;
	printf("\n\nEste programa determina o volume de n cubos, e determina o valor soma total.");
	printf("\n\nDigite quantos cubos voce deseja calcular o volume: ");
	scanf("%d", &n);
	for (i = 1; i <= n; i++){
		printf("\n\n\n\nDigite o valor da aresta do %d cubo: ", i);
		scanf("%d", &aresta);
		volume = aresta * aresta * aresta;
		soma = soma + volume;
		printf("\nO valor volume do %d cubo e de: %d.", i, volume);
	}
	printf("\n\n\n\nO valor total dos %d cubos e de: %d.", n, soma);
	printf("\nObrigado por utilizar nosso sistema!");
	return 0;
}

File: exercicios em Python/q81.py
import os

print("\n\nEste programa determina o volume de n cubos, e determina o valor soma total.")
n = int(input("\n\nDigite quantos cubos voce deseja calcular o volume: "))
i = 1
soma = 0
volume = 0
while i <= n:
        print("\n\n\n\nDigite o valor da aresta do ",i," cubo: ")
        aresta = float(input(""))
        volume = int(aresta * aresta * aresta)
        soma = int(soma + volume)
        print("\nO valor volume do ",i," cubo e de: ",volume,".")
        i = i + 1

print("\n\n\n\nO valor total dos ",n," cubos e de: ",soma,".")
print("\nObrigado por utilizar nosso sistema!")
os.system("pause")
